src/desktop/components/artwork_filter_2/templates/index.jade

```diff
@@ -5,10 +5,21 @@
 
   #artwork-filter-right
     #artwork-filter-right__header
-      #artwork-filter-right__totals
-        //- Rendered separately
-      #artwork-filter-right__sorts-dropdown
-        //- Rendered separately
+
+      //- FIXME: Replace with proper A/B test
+      //- if sd.ARTIST_PAGE_PAGINATION is 'control'
```

Comment:
should the commented out code be `is 'experiment'`? Since this is the pagination stuff, which _is_ the experiment, right? I might be confused!